components/Forms/LoginForm.vue

```diff
@@ -102,8 +102,15 @@ export default Vue.extend({
 
       this.loading = true;
       this.setDeviceProfile();
-      await this.loginRequest(this.login);
-      this.loading = false;
+      try {
+        await this.loginRequest(this.login);
+        this.$router.replace('/');
```

Comment:
Why replace instead of push?

Reply:
So there's no way to go back. Doesn't make much sense if you're already logged in.